woocommerce/woocommerce-ios
# [Mobile Payments] Receipts - Ensure receipt is fit to the page properly

Closes: #4265

@ctarda @joshheald - just one reviewed needed, but both of y'all are welcome

### Description
- On the Brother RJ-4250BW printer, the right edge of the receipt would be cut off on the first printing. Worse, subsequent printings would have the receipt width correct but the printer was ejecting meters of receipt paper for each receipt.
- As an interim measure, until we can determine how to confidently calculate the height of the receipt content for a given page width, we're going to constrain the receipt page size to 4 inches by 10 inches. For roll printers like the RJ-4250BW, we are also using the AirPrint cut delegate to set the page (cut) height to 11 inches.
- In the future, I think we should consider _different_ output formats for roll printers and sheet printers. A sheet printer should probably leverage headers and footers and page numbers for multi-page receipts for example.

### Testing instructions
- Select the RJ-4250BW and print a receipt. Ensure the right edge is NOT cut off (you should see the line item totals) and that the receipt fits on a single "page" of about 11 inches of paper.
- Repeat for an inkjet printer (e.g. a Canon Pixma MG6220). Ensure the receipt fits to the page as well (note that the receipt will still only be 4 inches wide even when printed on A4 or US Letter paper.

### Screenshots

Brother Roll Printer
<img src="https://user-images.githubusercontent.com/1595739/144150849-a7e1404f-6bb8-4910-86eb-52ea8d52e972.png" width=70%/>

<img src="https://user-images.githubusercontent.com/1595739/144150844-86e0a825-2076-49fe-9065-6bfb76482eed.png" width=70%/>

<img src="https://user-images.githubusercontent.com/1595739/144151192-49611f20-8af6-48dc-8837-10e2ae3df963.jpg" width=70%/>

---
